index.js

```diff
@@ -0,0 +1,102 @@
+let countriesData = [];  
+const countriesGrid= document.querySelector('.countries-grid');
+const searchInput= document.querySelector('.search-input');
+const dropdownWrapper = document.querySelector('.dropdown-wrapper');
+const dropdownHeader = document.querySelector('.dropdown-header');
+
+
+fetch('./CountriesData.json')
+  .then((response) => response.json())
```

Comment:
Hardcoded strings should be saved inside constant file, this will enable you to add or change things inside 1 place (Other places might need this string as well)